golem_messages/message/tasks.py

```diff
@@ -89,6 +91,14 @@ def __init__(self, header: datastructures.MessageHeader = None,
         if self.concent_enabled is None:
             self.concent_enabled = True
 
+    @property
+    def requestor_ethereum_address(self):
+        return '0x{}'.format(self.requestor_ethereum_public_key[:20].hex())
```

Comment:
This isn't correct. The right formula is `address = sha3(public_key)[12:]`. See `pubkeytoaddr` in `golem.utils`